tests/sgx/extra_data/host/host.c

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <assert.h>
+#include <malloc.h>
+#include <openenclave/bits/sgx/sgxtypes.h>
+#include <openenclave/host.h>
+#include <openenclave/internal/sgx/extradata.h>
+#include <openenclave/internal/sgx/tests.h>
+#include <openenclave/internal/tests.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "extra_data_u.h"
+
+#define SKIP_RETURN_CODE 2
+
+static oe_result_t _load_extra_enclave_data_hook(
+    oe_load_extra_enclave_data_hook_arg_t* arg,
+    uint64_t baseaddr)
+{
+    __attribute__((__aligned__(4096))) uint8_t page[OE_PAGE_SIZE];
```

Comment:
This uses a GCC define in a host (not enclave) file.   It would be better to be compiler agnostic (e.g., be capable of working with MSVC) by using a macro here that can be specified per compiler instead of hard coding `__attribute__` inline.  openenclave\bits\defs.h already has a bunch of such macros for attributes.  Please add another one there and use the macro here.